src/physics/longwave_radiation.jl

```diff
@@ -123,12 +123,12 @@ function longwave_radiation!(
     # extension to Jeevanjee: Include temperature flux between surface and lowermost air temperature
     # but zero flux if land/sea not available
     Fₖ_ocean = isfinite(skin_temperature_sea) ?
-        (T[end] - skin_temperature_sea) * α * (Tₜ - skin_temperature_sea) : 
+        (T[end] - skin_temperature_sea) * 3.1 * α * (Tₜ - skin_temperature_sea) : 
```

Comment:
The continuous integration tests for differentiability with Enzyme are failing because this change introduces a type instability. `3.1` is a `Float64`, but we have `NF` (so e.g. `Float32`) precision here. So `Fₖ_ocean` can be either a `Float64` or `NF`.  

An easy fix should be to just write `NF(3.1)` here instead of `3.1`.